browserstack.err

```diff
@@ -0,0 +1 @@
+[object Object]
```

Comment:
This should be in .gitignore. Can you create a PR against master to remove it from master too, if present? I'm not sure how/when this was generated